lessor.go

```diff
@@ -343,6 +345,12 @@ func run() error {
 		log.Print("[ERROR] Negative of zero amount to lease")
 		return errFailure
 	}
+	if leasingThreshold > 0 {
```

Comment:
If you are going to keep leasingThreshold `integer`, should we check `if leasingThreshold < 0` ?

Reply:
Nope, only positive values are meaningful.